### PR TITLE
RATEST-275:NEW E2E Test: Manage OCL Subscription Workflow

### PR DESCRIPTION
**Pull request by jwnasambu**
**Issue worked on:**

https://issues.openmrs.org/browse/RATEST-275

**Description of what changed:**

### CODE REVIEW

File: src/test/resources/org/openmrs/uitestframework/test.properties [hunk 1]
```diff
@@ -4,5 +4,5 @@ login.password=Admin123
 login.location=Pharmacy
 webdriver=firefox
 login.auto=false
-headless=false
-webdriver.gecko.driver=firefoxdriver/mac/geckodriver
\ No newline at end of file
+headless=true
```
Comment (sherrif10) [inline src/test/resources/org/openmrs/uitestframework/test.properties]: You are supposed not to change these file, Is this failing without these change.

Comment (jwnasambu) [reply src/test/resources/org/openmrs/uitestframework/test.properties]: @sherrif10 I have already fixed the mistake I did. Thanks for the review though!

**Comment by kdaud** [comment]
@jwnasambu some merge conflicts need to be resolved.

File: src/test/java/org/openmrs/contrib/qaframework/automation/AdvancedInstallSteps.java [hunk 2]
```diff
@@ -9,16 +9,16 @@
...
-import org.openmrs.contrib.qaframework.helper.InitialSetupPage;
-
 import io.cucumber.java.After;
 import io.cucumber.java.Before;
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 
+import org.openmrs.contrib.qaframework.RunTest;
```
Comment (sherrif10) [inline src/test/java/org/openmrs/contrib/qaframework/automation/AdvancedInstallSteps.java:19]: is this change neccessary here

File: src/test/java/org/openmrs/contrib/qaframework/automation/UpgradeSteps.java [hunk 3]
```diff
@@ -9,14 +9,14 @@
...
 
-import org.openmrs.contrib.qaframework.RunTest;
-
 import io.cucumber.java.After;
 import io.cucumber.java.Before;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 
+import org.openmrs.contrib.qaframework.RunTest;
```
Comment (sherrif10) [inline src/test/java/org/openmrs/contrib/qaframework/automation/UpgradeSteps.java:18]: You can include changes that are only required to make reveiwing qiute simpler

File: src/test/java/org/openmrs/contrib/qaframework/automation/AdvancedInstallSteps.java (hunk 2 above)
Comment (jwnasambu) [reply src/test/java/org/openmrs/contrib/qaframework/automation/AdvancedInstallSteps.java:19]: @sherrif10 I don't remember making changes on that file.

**Pull request closed without merging**